app/assets/stylesheets/components/_timeline.scss

```diff
@@ -1,11 +1,15 @@
+/* ===================================================== *\
+   Timeline feature (originally in draft MMT progress page)
```

Comment:
I was going to leave this alone but since I think you have to make one more change anyways, can we also specify this was originally for proposals?